src/shared/multiStepInputFlowController.ts

```diff
@@ -123,10 +122,8 @@ export class MultiStepInputFlowController {
                     input.onDidTriggerButton(async item => {
                         if (item === QuickInputButtons.Back) {
                             reject(InputFlowAction.back)
-                        } else if (item instanceof HelpButton) {
-                            if (item.url) {
-                                await env.openExternal(Uri.parse(item.url))
-                            }
+                        } else if ((item as any).onClick !== undefined && typeof (item as any).onClick === 'function') {
```

Comment:
typeof works on undefined, so I guess the first condition can be omitted
```suggestion
                        } else typeof (item as any).onClick === 'function') {
```